common-util/prop-types.js

```diff
@@ -1,20 +1,59 @@
 import PropTypes from 'prop-types';
 
+const Voter = PropTypes.shape({
+  address: PropTypes.string,
+  signature: PropTypes.string,
+  votingPower: PropTypes.string, // passed in wei
```

Comment:
Not for now, but I think this key would be clearer if it was called `veOlasInWei`. Would require sync with @dvilelaf as well.